tests/clients/test_job.py

```diff
@@ -47,6 +58,10 @@
     SubrunSummarySchema,
 )
 
+MILLI_CPUS = 0.1
```

Comment:
`MILLI_CPUS` should be an int -  a better example value would be 1000.

Reply:
Could also just inline this and `MEMORY_MB` in `INSTANCE_SIZE` below as it's only used there.

Similar for `MAX_RUNTIME_SECONDS` in `JOB_DEFINITION`/